static/styles/zulip.css

```diff
@@ -3896,6 +3916,11 @@ form.admin-realm .control-label {
         margin-right: 7px;
     }
 
+    #subscriptions-status {
+        left:250px;
+        width:50%;
```

Comment:
these still don't have correct whitespace; there should be a spaec between the : and the number. (issue is still present below too)